bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Recaptcha.java

```diff
@@ -0,0 +1,37 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2023 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ConsumerType;
+
+/**
+ * Defines the form {@code reCaptcha} Sling Model used for the {@code /apps/core/fd/components/form/captcha/v1/captcha}
+ * component.
+ *
+ * @since com.adobe.cq.forms.core.components.models.form 2.0.0
+ */
+@ConsumerType
+public interface Recaptcha extends Field {
+
+    default String getrcCloudServicePath() {
+        return null;
+    }
+
+    default String getRecaptchaSize() {
```

Comment:
`getSize`

Reply:
This is in consonance with property name recaptchaSize. Renaming this to 'size' was not allowing it to get mapped correctly from the value configured in edit dialog since a size and getSize attribute and method already exists elsewhere.

Reply:
we can always have the mapping with different name in jackson. no need to couple the interface with persistence.